src/components/CompareResults/RevisionSelect.tsx

```diff
@@ -41,6 +41,7 @@ function RevisionSelect() {
         defaultValue=''
         displayEmpty
         size='small'
+        inputProps={{ 'aria-label': 'Filter by revision' }}
```

Comment:
Without this new prop, the select didn't have any label anymore, and we couldn't target it in tests. This is also better for accessibility.